core/domain/obj_services_test.py

```diff
@@ -43,8 +43,8 @@ def test_base_object_is_not_gettable(self):
 
 class ObjectDefaultValuesUnitTests(test_utils.GenericTestBase):
```

Comment:
I think we should move this test to appropriate places as `get_default_object_values` doesn't exist anymore. Can you move this to constants_test.py with appropriate test name? (You can find guidelines on writing tests over [here](https://github.com/oppia/oppia/wiki/Writing-backend-tests#guidelines-for-writing-good-tests))

Reply:
I would argue that the tests should remain in `obj_services_test.py`, but if you believe otherwise, I could move it. I believe we should keep the `ObjectRegistry` tests with `ObjectDefaultValues` tests.

Reply:
Looking at this, I suggest leaving the second test here (since it's related to obj_registry) but moving the first test to extensions/interactions/base_test.py (since that's about all the interactions). Would that work?

Reply:
Sounds good to me! will it be fine @esyang97?